sklearn_bench/knn_clsf.py

```diff
@@ -39,7 +37,8 @@
                     help='Distance metric to use')
 params = bench.parse_args(parser)
 
-from sklearn.neighbors import KNeighborsClassifier
+if not params.no_intel_optimized:
+    from sklearn.neighbors import KNeighborsClassifier

```

Comment:
It is not clear why estimators are imported only in one case

Reply:
Also done